tools/bumbershoot/myrimatch/myrimatch.xml

```diff
@@ -0,0 +1,241 @@
+<tool id="myrimatch" version="0.1.0" name="MyriMatch">
+    <description></description>
+    <requirements>
+        <requirement type="package" version="3.0.8789">bumbershoot</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" level="fatal" description="Job Failed" />
+        <regex match="^Could not find the default configuration file.*$"
+          source="both"
+          level="warning" />
+    </stdio>
+    <command>
+        #set $db_name = $input_database.display_name.replace(".fasta", "") + ".fasta"
+        #if $output_type.value == "mzid"
+            #set $output_ext="mzid"
+            #set $output_format="mzIdentML"
+        #else
+            #set $output_ext="pepXML"
+            #set $output_format="pepXML"
+        #end if
+        #set $input_name = $input.display_name
+        #set $output_name = $input_name.split(".")[0] + "." + $output_ext
+
+        #set $static_mods_str = ""
+        #for $static_mod in $static_mods
+          #set $static_mods_str = $static_mods_str + " " + str($static_mod.aa) + " " + str($static_mod.mass)
+        #end for
+        #set $static_mods_str = $static_mods_str.lstrip()
+
+        #set $dynamic_mods_str = ""
+        #for $dynamic_mod in $dynamic_mods
+            #set $dynamic_mods_str = $dynamic_mods_str + " " + str($dynamic_mod.motif) + " * " + str($dynamic_mod.mass)
+        #end for
+        #set $dynamic_mods_str = $dynamic_mods_str.lstrip()
+
+        ln -s '$input' '${input_name}';
+        ln -s '$input_database' '${db_name}';
+        nice myrimatch -cpus 1 -DecoyPrefix '${decoy_prefix}'
```

Comment:
How do I do that? By default MyriMatch will take all the CPUs it can see. With my test setup, that meant all of them. So if I was running multiple jobs, the docker-galaxy-stable would run 3 of those, each of which would run 8 threads, with default priority.